contracts/lease/src/api/open/mod.rs

```diff
@@ -112,23 +117,111 @@ impl InterestPaymentSpec {
     }
 }
 
-#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, JsonSchema)]
+#[derive(Serialize, Clone, PartialEq, Eq, JsonSchema)]
 #[cfg_attr(any(test, feature = "testing"), derive(Debug))]
 #[serde(deny_unknown_fields, rename_all = "snake_case")]
 pub struct PositionSpec {
+    /// Liability constraints
     pub liability: Liability,
+    /// The minimum amount to liquidate. Any attempt to liquidate a smaller
+    /// amount would be postponed until the amount goes above this limit
     pub min_asset: LpnCoin,
+    ///  The minimum amount that a lease asset should be evaluated past any
+    ///  partial liquidation or close. If not, a full liquidation is performed
     pub min_sell_asset: LpnCoin,
 }
 
 impl PositionSpec {
-    #[cfg(any(test, feature = "testing"))]
     pub fn new(liability: Liability, min_asset: LpnCoin, min_sell_asset: LpnCoin) -> Self {
```

Comment:
it's intended to be used only for DO-to-DTO conversion internally, so it's better to reduce its visibility to `pub(crate)`